views/result_selection.py

```diff
@@ -103,22 +125,54 @@ def remove_selected_ts_ds(self):
             self.ts_datasource.removeRows(row,1)
 
     def get_3di_spatialites_legendlist(self):
+        """
+        get list of spatialite data sources currently active in canvas
```

Comment:
"Get" => "Return"

(En in principe ben ik wat allergisch voor methodes met "get_" in de naam, da's vaak eerder java-achtig dan python-achtig, dus kijk even of dat de handigste naam is.)